tools/harness-thci/OpenThread.py

```diff
@@ -2963,6 +2963,10 @@ def setBbrDataset(self, SeqNumInc=False, SeqNum=None, MlrTimeout=None, ReRegDela
             False: fail to set BBR Dataset
         """
         assert not (SeqNumInc and SeqNum is not None), "Must not specify both SeqNumInc and SeqNum"
+
+        if MlrTimeout and MlrTimeout != self.bbrMlrTimeout or ReRegDelay and ReRegDelay != self.bbrReRegDelay:
+            SeqNumInc = True
+
         if SeqNumInc:
             if self.bbrSeqNum in (126, 127):
                 self.bbrSeqNum = 0
```

Comment:
looks like `SeqNum` parameter is not handled. 

Should there be a line in the else part?
`self.bbrSeqNum = SeqNum`

Reply:
You are right, good catch!

Reply:
Interesting. We didn't handle `SeqNum` parameter?

Reply:
I believe this bug was introduced recently.